mozregression/test_runner.py

```diff
@@ -196,7 +196,7 @@ def evaluate(self, build_info, allow_back=False):
 
             env = dict(os.environ)
             for k, v in variables.iteritems():
-                env['MOZREGRESSION_' + k.upper()] = v
+                env['MOZREGRESSION_' + k.upper()] = v.encode('ascii', errors='ignore')
```

Comment:
In line 192 there is already a call to `str()`. I think that one should be removed in favor of the conversion you just added.

Reply:
You are right, it looks like the error we're seeing must have some other root cause. :( Since if the value was unicode we would have seen an exception

Reply:
The only value we do not explicitly convert into a string is the binary. Could we have a strange value here?

Reply:
Seems unlikely but possible I suppose?

Reply:
Maybe we can just do what you suggest and hope for the best. :)